scripts/bulk_update_address_countries.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import logging
+from closeio_api import Client as CloseIO_API
+
+LEADS_QUERY = '* sort:created'
+
+parser = argparse.ArgumentParser(description='changing old country code to new country code')
+parser.add_argument('old_code', type=str, help='Old country code')
+parser.add_argument('new_code', type=str, help='New country code')
```

Comment:
should trigger a warning if either old_code or new_code is something other than a 2 character string
should also mention in the description that the code should be an iso country code